api.rst

```diff
@@ -1227,6 +1229,30 @@ as in ``{1,2,3,4}``. Note that the curly brackets have to be urlencoded(``{`` is
 
    In these versions we recommend using function parameters of type json to accept arrays from the client.
 
+.. _s_procs_variadic:
```

Comment:
Hey @wolfgangwalther. Since we'll not include variadic in this release, could you leave this section for another PR so I can merge?

Recently, we had a user confused about volatility on [gitter chat](https://gitter.im/begriffs/postgrest?at=5f22fba3a050df430025ff2a). This addition to the docs would clear things up.